packages/contracts/src/dollar/facets/BancorFormulaFacet.sol

```diff
@@ -0,0 +1,69 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.16;
+
+import {LibBancorFormula} from "../libraries/LibBancorFormula.sol";
+import {Modifiers} from "../libraries/LibAppStorage.sol";
+
+/**
+ * @title Bancor formula
+ * @dev Modified from the original
+ * @notice forked from Bancor:
+ * https://github.com/bancorprotocol/contracts
+ *
+ */
+contract BancorFormulaFacet is Modifiers {
```

Comment:
same contract name inside facets/bancorformulaFacet

Reply:
Do you want the name modified?